md/data-management.md

```diff
@@ -0,0 +1,34 @@
+# Data Management in the UI Designer
+
+Discover how to explore [Business Data Model](define-business-data-model.md) (BDM) objects in the UI Designer, and use it to create user interfaces and variables.
+
+A **Data Model** tab is available from the Palette of the Page editor:
+
+![data_model_panel](images/data_model_panel.png)
+
+## Exploring the BDM
+From the Data Model tab, the graph icon allows you to visualize and explore the BDM.
```

Comment:
Do you think it's a good idea to say here: "we use Voyager tool"? It's can be make some traffic on google if someone search voyager tool, we can guess we are in the result ?

Reply:
Not sure we want to mention it, as discussed we may change the tool. Documenting it is a kind of commitment with it...